math/test.cpp

```diff
@@ -10,3 +12,7 @@ TEST_CASE("plus operator on ints", "[plus]") {
 
 // Add more test cases down here
 
+vector<int> v{1, 2, 3, 4};
```

Comment:
Good job getting started but we don't want to declare variables outside of test cases.